frontend/src/store/index.js

```diff
@@ -309,6 +309,20 @@ export const store = new Vuex.Store({
       return false
     },
 
+    is_liaison_designate(state) {
```

Comment:
Why are these here? Are they not already defined like 10 lines up?

Reply:
merge conflict resolution gone awry...